app/views/interactive_pages/_interactive.html.haml

```diff
@@ -1,5 +1,5 @@
 -# Do not pin multiple interactives, as it means that the user might not be able to see the bottom ones
 -# if his screen is shorter that the height of all the interactives
-.interactive-mod{class: (page.visible_interactives.count > 1 || layout == 'l-full-width') ? nil : 'pinned'}
+.interactive-mod{class: layout == 'l-full-width' ? nil : 'pinned'}
```

Comment:
Probably should update the comment above this lines so it matches the code.

Reply:
Thanks, fixed.